.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.zsh-theme text eol=lf
+*.zsh text eol=lf
+*.sh text eol=lf
```

Comment:
```
# Set the default behavior, in case people don't have core.autocrlf set.
* text=auto

# Explicitly declare text files you want to always be normalized and converted
# to native line endings on checkout.
*.zsh-theme text
*.zsh text
*.sh text
```

Reply:
Reference: https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings

Reply:
Added some comments for clarification. Let me know if there's something more specific that you're looking for.